File: SRC - PERL/00 - Estruturas Sequenciais/23.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba o custo de um espetáculo teatral e o preço do convite desse espetáculo. Esse programa deverá calcular e mostar a quantidade de convites que devem ser vendidos para que pelo menos o custo do espetáculo seja alcançado

print "Digite o custo do espetáculo > ";
my $custo_espetaculo = <>;
chomp $custo_espetaculo;

print "Digite o preço do convite > ";
my $preco_convite = <>;
chomp $preco_convite;

my $quantidade = $custo_espetaculo / $preco_convite;

printf "A quantidade que deve ser vendida é %d \n", $quantidade;

File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/23.py
#!/usr/bin/python
# coding: latin-1
# 23 - Faça um programa que receba o custo de um espetáculo teatral e o preço do convite desse espetáculo. Esse programa deverá calcular
# e mostar a quantidade de convites que devem ser vendidos para que pelo menos o custo do espetáculo seja alcançado

custo = float(input("Digite o custo do espetáculo teatral: "))
convite = float(input("Digite o custo do convite para o espetáculo: "))

qtd = custo / convite
qtd = round(qtd)

print("Deverão ser vendidos", qtd, "convites para que o custo do espetáculo seja alcançado.")
